backend/functions/src/student/functions.ts

```diff
@@ -24,6 +24,17 @@ async function removeStudentFromCourse(
   }
 }
 
+const validateEmail = async (email: string) => {
+  const emailRegex = /^\w+@cornell.edu$/
+  // if true => valid email. if false => invalid email.
+  const validEmail = emailRegex.test(email)
+  console.log(validEmail)
+  if (validEmail === false) {
+    throw new Error('Invalid Email Form')
+  }
```

Comment:
I think we should avoid using `console.log` in backend code now because we can use the new logger functions. Is this important to log, without any context?

I feel like `if (validEmail === false)` can be written more idomatically

Reply:
Ok yeahhh, I don't think my brain was working tbh. Thanks for the tips, I think I implemented it a bit better now?

Put it inside the singular function, the condition, and removed the console logging (I forgot to remove it initially) Lmk if there's still anything else.